src/sentry/snuba/discover.py

```diff
@@ -192,27 +192,24 @@ def find_histogram_buckets(field, params, conditions):
     )
     if len(results["data"]) != 1:
         # If there are no transactions, so no max duration, return one empty bucket
-        return "histogram({}, 1, 1)".format(column)
+        return "histogram({}, 1, 1, 0)".format(column)
 
-    bucket_max = results["data"][0][max_alias]
     bucket_min = results["data"][0][min_alias]
+    bucket_max = results["data"][0][max_alias]
     if bucket_max == 0:
         raise InvalidSearchQuery(u"Cannot calculate histogram for {}".format(field))
     bucket_size = ceil((bucket_max - bucket_min) / float(num_buckets))
-    if bucket_size <= 1.0:
-        raise InvalidSearchQuery(
-            u"Cannot calculate histogram for {}. Resulting buckets are too small.".format(field)
-        )
+    offset = floor(bucket_min / bucket_size)
```

Comment:
Is there a test for this? I don't understand why the offset wouldn't be equal to the bucket_min. Why do you divide by the bucket size?